src/main/java/calculator/StringHandler.java

```diff
@@ -0,0 +1,21 @@
+package calculator;
+
+
+public class StringHandler {
+
+    public int checkString(String str) {
+
+        CalculateString calculateString = new CalculateString();
+        if (str.trim().isEmpty()) {
+            return 0;
+        }
+        if (str.startsWith("//")) {
+            return calculateString.customString(str);
+
```

Comment:
```suggestion
        if (str.startsWith("//")) {
            return calculator.calculate(expression);

```

자바에서 메서드는 특정 동작을 수행하기 때문에, 메서드명은 동사로 시작하도록 작성하는 편이에요.
또한 `calculateString.customString(str)` 이 부분만 봐도 코드가 어떤 역할을 하는지 알아보기 편하도록 작성해보면 어떨까요?